test/pkg/environment/common/expectations.go

```diff
@@ -294,6 +294,19 @@ func (env *Environment) EventuallyExpectTerminatingWithTimeout(timeout time.Dura
 	}).WithTimeout(timeout).Should(Succeed())
 }
 
+func (env *Environment) eventuallyExpectNoLeakedKubeNodeLease() {
+	GinkgoHelper()
+	Eventually(func(g Gomega) {
+		// expect no kube node lease to be leaked
+		leases := &coordinationv1.LeaseList{}
+		g.Expect(env.Client.List(env.Context, leases, client.InNamespace("kube-node-lease"))).To(Succeed())
+		leakedLeases := lo.Filter(leases.Items, func(l coordinationv1.Lease, _ int) bool {
+			return l.OwnerReferences == nil
+		})
+		g.Expect(len(leakedLeases)).To(BeNumerically("==", 0))
```

Comment:
```suggestion
		g.Expect(leakedLeases).To(HaveLen(0))
```

nit: Makes it print cleaner when it fails